src/quicklist.c

```diff
@@ -1635,6 +1565,39 @@ void quicklistPush(quicklist *quicklist, void *value, const size_t sz,
     }
 }
 
+/* Print info of quicklist which is used in debugCommand. */
+void quicklistRepr(unsigned char *ql, int full) {
+    int i = 0;
+    quicklist *quicklist  = (struct quicklist*) ql;
+    printf("{count : %ld}\n", quicklist->count);
```

Comment:
This PR moves this function from after quicklistGetLzf to after quicklistPush. Is there a reason for this?

I think it's easier to review if the function is not moved. For example, the first lines of this functions are not actually changed.

Reply:
this was done to solve a bug in https://github.com/redis/redis/pull/9740/commits/7987b9b8dea7018237f3fdf750a907e18e5e5834
Sadly since quicklistRecompressOnly is a macro we can't add a forward declaration, so one of them has to be moved. 

On the bright side, it's a completely new function, so this blame log change isn't gonna cause any conflicts.

Reply:
Ok